jaseci_ai_kit/utils/test_runner.py

```diff
@@ -0,0 +1,73 @@
+import subprocess
```

Comment:
What's the purpose of this script? Do we need it for the this PR?

Reply:
We dont need it